metrics/format_json.py

```diff
@@ -4,7 +4,7 @@
 import json
 
 data = json.load(sys.stdin)
-sep = "{\n"
+sep = "module.exports = {\n"
```

Comment:
Looks like you forgot to add the corresponding semicolon at the end of the file, which is why the linter is failing? That should go on line 16, but github isn't letting me add a comment there.